packages/office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx

```diff
@@ -124,7 +177,7 @@ export class ListScrollingExample extends React.Component<IListScrollingExampleP
   private _onRenderCell = (item: IExampleItem, index: number): JSX.Element => {
     return (
       <div data-is-focusable={true}>
-        <div className={css(styles.itemContent, index % 2 === 0 ? styles.itemContentEven : styles.itemContentOdd)}>
+        <div className={css(classNames.itemContent, index % 2 === 0 ? classNames.itemContentEven : classNames.itemContentOdd)}>
```

Comment:
As part of this work @pandasa123 would you be open to moving the even / odd logic into the CSS itself similar to the following?

```css
'li:nth-child(even)'
```
in https://github.com/OfficeDev/office-ui-fabric-react/pull/9537/files#diff-9c15ec532524b9b7d35506b4245caebdR27 as an example.

Reply:
Hey Kevin! I was looking into that and I could but it's scoped by the class rather than tag. Since there's no `nth-class`-esque, it might take a bit more refactoring but up to you!

Reply:
Ah right, good point. I will follow-up separately :).